packages/edit-post/src/index.js

```diff
@@ -78,6 +80,22 @@ export function initializeEditor(
 	settings,
 	initialEdits
 ) {
+	// Prevent adding template part in the post editor.
+	// Only add the filter when the post editor is initialized, not imported.
+	addFilter(
```

Comment:
I wonder if this should be inside a "useEffect" call as this is probably going to add a new filter on each render.

Reply:
That's what I initially did, only to find we're not inside of a React component here :D There's another listener registered in this function, a few dozen lines below. I believe this function is only called once:

https://github.com/WordPress/gutenberg/blob/93bd8e9d0df9346a6923424f8135570be1bcbf36/lib/full-site-editing/edit-site-page.php#L206

https://github.com/WordPress/gutenberg/blob/93bd8e9d0df9346a6923424f8135570be1bcbf36/lib/editor-settings.php#L87-L98

```html
<script id='wp-edit-post-js-after'>
( function() {
 window._wpLoadBlockEditor = new Promise( function( resolve ) {
   wp.domReady( function() {
     resolve( wp.editPost.initializeEditor( 'editor', // ...
   } );
 } )();

```

Reply:
make sense 👍